### PR TITLE
Module1 cycles task2 (Function values on the interval  a,b with step h)

### PR DESCRIPTION
**Pull request by pmiakish**

### CODE REVIEW

File: module_1/src/com/epam/cycles/CyclesTaskB.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+                for (double x = a; x <= b; x += h) {
+                    if (x <= CONDITION_PARAMETER) {
+                        System.out.println ("y = " + (x * -1.0) + " (x = " + x + ")");
+                    } else {
+                        System.out.println ("y = " + x + " (x = " + x + ")");
+                    }
+                }
+            } else {
+                System.out.println ("Incorrect interval [a,b] or step value (h)! Try again...");
+            }
```
Comment (lovelyserafima) [inline module_1/src/com/epam/cycles/CyclesTaskB.java:30]: break this habit of adding tabs for a new block of code :)

File: module_1/src/com/epam/cycles/CyclesTaskB.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+    public static void main(String[] args) {
+
+        Scanner console = new Scanner(System.in);
+        System.out.println ("Set the interval [a,b]:");
+        System.out.print (" -- enter the value 'a': ");
+        double a = console.nextDouble();
+        System.out.print (" -- enter the value 'b': ");
+        double b = console.nextDouble();
+        System.out.print ("Set the step value (h): ");
```
Comment (lovelyserafima) [inline module_1/src/com/epam/cycles/CyclesTaskB.java:16]: Still spaces after System.out.println
```suggestion
        System.out.println("Set the interval [a,b]:");
        System.out.print(" -- enter the value 'a': ");
        double a = console.nextDouble();
        System.out.print(" -- enter the value 'b': ");
        double b = console.nextDouble();
        System.out.print("Set the step value (h): ");
```

File: module_1/src/com/epam/cycles/CyclesTaskB.java [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        double h = console.nextDouble();
+
+            if (a < b && b - a >= h) {
+                System.out.println ("Function values on the interval [" + a + ", " + b + "]:");
+                for (double x = a; x <= b; x += h) {
+                    if (x <= CONDITION_PARAMETER) {
+                        System.out.println ("y = " + (x * -1.0) + " (x = " + x + ")");
```
Comment (lovelyserafima) [inline module_1/src/com/epam/cycles/CyclesTaskB.java:23]: I think it is more beautiful to write 
```suggestion
                        System.out.println ("y = " + -x + " (x = " + x + ")");
```
or at least 
```suggestion
                        System.out.println ("y = " + (-x) + " (x = " + x + ")");
```

File: module_1/src/com/epam/cycles/CyclesTaskB.java [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+        Scanner console = new Scanner(System.in);
+        System.out.println ("Set the interval [a,b]:");
+        System.out.print (" -- enter the value 'a': ");
+        double a = console.nextDouble();
+        System.out.print (" -- enter the value 'b': ");
+        double b = console.nextDouble();
+        System.out.print ("Set the step value (h): ");
+        double h = console.nextDouble();
+
+            if (a < b && b - a >= h) {
```
Comment (lovelyserafima) [inline module_1/src/com/epam/cycles/CyclesTaskB.java:19]: I think you need also check that h > 0

File: module_1/src/com/epam/cycles/CyclesTaskB.java (hunk 1 above)
Comment (pmiakish) [reply module_1/src/com/epam/cycles/CyclesTaskB.java:30]: I'll try

File: module_1/src/com/epam/cycles/CyclesTaskB.java (hunk 3 above)
Comment (pmiakish) [reply module_1/src/com/epam/cycles/CyclesTaskB.java:23]: didn't think it worked like that

File: module_1/src/com/epam/cycles/CyclesTaskB.java (hunk 4 above)
Comment (pmiakish) [reply module_1/src/com/epam/cycles/CyclesTaskB.java:19]: a good idea

File: module_1/src/com/epam/cycles/CyclesTaskB.java (hunk 1 above)
Comment (lovelyserafima) [reply module_1/src/com/epam/cycles/CyclesTaskB.java]: You did it!

File: module_1/src/com/epam/cycles/CyclesTaskB.java (hunk 3 above)
Comment (lovelyserafima) [reply module_1/src/com/epam/cycles/CyclesTaskB.java]: You always can try!

**Pull request merged**